examples/aws/monitor/vars.tf

```
@@ -12,6 +12,14 @@ variable "monitor" {
   default = {}
 }
 
+variable "target_monitoring" {
```

```suggestion
variable "targets" {
```
It's a variable in monitor module so `monitoring` is probably redundant.

---

Fixed in 55b8e78